pandas/tseries/resample.py

```diff
@@ -652,9 +652,6 @@ def _convert_obj(self, obj):
             # Cannot have multiple of periods, convert to timestamp
             self.kind = 'timestamp'
 
-        if not len(obj):
-            self.kind = 'timestamp'
-
```

Comment:
Am going to punt this to https://github.com/pydata/pandas/issues/12774, and loosen the associated test in this PR